### PR TITLE
[FTX]Adding FTX fundingPayments and borrowingHistory support

### PR DESCRIPTION
**Pull request by makarid**
Also, adding a starting point for ftx-stream

### CODE REVIEW

File: xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxAccountServiceRaw.java [hunk 1]
```diff
@@ -95,4 +95,33 @@ public FtxAccountServiceRaw(Exchange exchange) {
...
+      return ftx.getFundingPayments(
+              exchange.getExchangeSpecification().getApiKey(),
+              exchange.getNonceFactory().createValue(),
+              signatureCreator,
+              subaccount,
+              startTime,
+              endTime,
+              future);
+    } catch (FtxException e) {
+      throw new FtxException(e.getMessage());
```
Comment (earce) [inline xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxAccountServiceRaw.java:124]: why don't you just throw e back?

**Comment by makarid** [comment]
Because i thought that we must a create an ExchangeException for every
exchange, everytime

Στις Κυρ, 7 Μαρ 2021 στις 9:50 μ.μ., ο/η Erick Arce <
notifications@github.com> έγραψε:

File: xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxAccountServiceRaw.java (hunk 1 above)
Comment (makarid) [reply xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxAccountServiceRaw.java:124]: For readability mostly

File: xchange-stream-ftx/pom.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <parent>
+        <groupId>org.knowm.xchange</groupId>
+        <artifactId>xchange-parent</artifactId>
+        <version>5.0.7-SNAPSHOT</version>
+    </parent>
+
+    <name>XChange Ftx Stream</name>
+    <artifactId>xchange-stream-ftx</artifactId>
+
+    <properties>
```
Comment (earce) [inline xchange-stream-ftx/pom.xml:15]: I don't think we want to target java11 right now @timmolter ?

Comment (makarid) [reply xchange-stream-ftx/pom.xml:15]: i am sorry my bad, i use Java 11 and i didn't change it. I will change it and push again

**Pull request merged**